vizro-core/tests/unit/vizro/tables/test_dash_table.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from vizro.tables.dash_table import _set_defaults_nested
+
+
+@pytest.fixture
+def default_dictionary():
+    return {"a": {"b": {"c": 1, "d": 2}}, "e": 3}
+
+
+@pytest.mark.parametrize(
+    "input, expected_output",
```

Comment:
```suggestion
    "input, expected",
```
I think this is our convention still?